### PR TITLE
update to timestamps

### PR DESCRIPTION
**Pull request by rstormsf**

### CODE REVIEW

File: contracts/Contribution.sol [hunk 1]
```diff
@@ -18,23 +18,23 @@ contract Contribution is Controlled, TokenController {
...
   uint256 public minimum_investment;
 
-  uint256 public startBlock;
-  uint256 public endBlock;
+  uint256 public startTime;
+  uint256 public endTime;
 
-  uint256 public initializedBlock;
-  uint256 public finalizedBlock;
+  uint256 public initializedTime;
```
Comment (Janther) [inline contracts/Contribution.sol]: For comunication with other contracts I also need `initializedBlock` and maybe `finalizedBlock`

Comment (rstormsf) [reply contracts/Contribution.sol]: but with which ones?

Comment (rstormsf) [reply contracts/Contribution.sol]: I already changed all variables in Exchange contract

**Comment by rstormsf** [comment]
Now I see. I thought we could completely move away from blocks counting. Looks like it's not the case right now since there are methods like TotalSupplyAt from MiniMe.

**Comment by rstormsf** [comment]
@Janther Please check it again

File: migrations/2_deploy_contracts.js [hunk 2]
```diff
@@ -1,9 +1,8 @@
-uvar MiniMeTokenFactory = artifacts.require("MiniMeTokenFactory");
```
Comment (Janther) [inline migrations/2_deploy_contracts.js:1]: lol

**Review by Janther — APPROVED** [review]

File: migrations/2_deploy_contracts.js (hunk 2 above)
Comment (rstormsf) [reply migrations/2_deploy_contracts.js:1]: :-)

**Pull request merged**